frontend/src/modules/entitieslist/components/EntitiesList.js

```diff
@@ -217,6 +217,12 @@ export class EntitiesListBase extends React.Component<InternalProps> {
         );
     };
 
+    getSiblingEntities = () => {
+        const { entity, locale } = this.props.parameters;
```

Comment:
We should retrieve data from parameters - the `entity` parameter reflects the entity that is currently open in the editor, but users could click on others.

You can get `locale` directly from props.

And you'll need to pass entity from the `Entity` component where you click on it. See `selectEntity` in Entity component for inspiration.